int-test/readme.md

```diff
@@ -0,0 +1,47 @@
+## Installation for the appium testing dependencies
+
+To install appium execute from the root folder of zulip-mobile
+```
+cd int-test
```

Comment:
We might want to add a script to that (maybe later, is good as instructions for now)